src/coffee/directives/api/models/child/marker-child-model.coffee

```diff
@@ -47,7 +47,6 @@ angular.module('uiGmapgoogle-maps.directives.api.models.child')
         else
           action = new PropertyAction (calledKey, newVal) =>
```

Comment:
Is this the actual change? Is so just PR marker-child-model.coffee. I will rebuild it post merge.